src/masternodes/mn_rpc.cpp

```diff
@@ -252,6 +252,14 @@ std::string ScriptToString(CScript const& script) {
     return EncodeDestination(dest);
 }
 
+int chainHeight(interfaces::Chain::Lock& locked_chain)
+{
+    LOCK(locked_chain.mutex());
+    if (auto height = locked_chain.getHeight())
+        return *height;
+    return 0;
+}
+
```

Comment:
That's old-fashioned, let's remove it, just use `view.GetLastHeight` instead

Reply:
Agreed. There's a lot of entries and on Saturday I was out of time so I restored this function. Will remove.